test/collection.test.js

```diff
@@ -7,140 +7,140 @@ const assert = require('assert');
 
 const mongoose = start.mongoose;
 
-describe('collections:', function() {
-  it('should buffer commands until connection is established', function(done) {
-    const db = mongoose.createConnection();
-    const collection = db.collection('test-buffering-collection');
-    let connected = false;
-    let insertedId = undefined;
-    let pending = 2;
-
-    function finish() {
-      if (--pending) {
-        return;
-      }
-      assert.ok(connected);
-      assert.ok(insertedId !== undefined);
-      collection.findOne({ _id: insertedId }).then(doc => {
-        assert.strictEqual(doc.foo, 'bar');
-        db.close();
-        done();
-      });
-    }
-
-    collection.insertOne({ foo: 'bar' }, {}, function(err, result) {
-      assert.ok(connected);
-      insertedId = result.insertedId;
-      finish();
+describe('collections:', function () {
```

Comment:
It looks like some linter or formatter made a lot of unnecessary changes in this. Can you please undo these unnecessary changes and just commit the tests you're adding without the extra formatting diff?

Reply:
Yep, sorry, should have run the linter before commit.